src/view/StrokeView.cpp

```diff
@@ -80,6 +80,8 @@ void StrokeView::drawNoPressure() {
         // Do not apply the alpha here, else the border and the fill
         // are visible instead of one homogeneous area
         DocumentView::applyColor(cr, s, 255);
+        // The operator will be reset (to CAIRO_OPERATOR_MULTIPLY) by cairo_pop_group_to_source below
+        cairo_set_operator(cr, CAIRO_OPERATOR_SOURCE);
```

Comment:
What if `group` is false?

Reply:
We're setting group to `true` on line 86. Moving this to just after line 79 might make this clearer:
```cpp
    cairo_push_group(cr);
    group = true;
    ... Lines 80-84 ...
    drawFillStroke();
}
```